skimage/transform/tests/test_geometric.py

```diff
@@ -544,6 +544,39 @@ def test_union_differing_types():
         tform1.__add__(tform2)
 
 
+@pytest.mark.parametrize(
+    "tform",
+    [
+        ProjectiveTransform(matrix=np.random.rand(3, 3)),
+        AffineTransform(scale=(0.1, 0.1), rotation=0.3),
+        EuclideanTransform(rotation=0.9, translation=(5, 5)),
+        SimilarityTransform(scale=0.1, rotation=0.9),
+        EssentialMatrixTransform(
+            rotation=np.eye(3), translation=(1 / np.sqrt(2), 1 / np.sqrt(2), 0)
+        ),
+        FundamentalMatrixTransform(
+            matrix=EssentialMatrixTransform(
+                rotation=np.eye(3), translation=(1 / np.sqrt(2), 1 / np.sqrt(2), 0)
+            ).params
+        ),
```

Comment:
Not sure, whether the style of initializing test data within the function or pytest parametrization is preferred. A downside of this is that the transforms are created at load time.

Reply:
It's a bit weird, but the entire test suite for this module isn't well structured (e.g. no grouping of tests by class). I'm not really happy but the same goes for alternatives I can think of.

Reply:
Yeah, I wouldn't worry too much about it.